### PR TITLE
add 'lib' symlink in tbb installation directory when building from source

### PR DESCRIPTION
**Pull request by mboisson**
Otherwise, embree does not find tbb

### CODE REVIEW

File: easybuild/easyblocks/t/tbb.py [hunk 1]
```diff
@@ -162,7 +162,6 @@ def install_step(self):
             self.libpath = os.path.join(self.libpath, libdir)
             self.log.debug("self.libpath: %s" % self.libpath)
             # applications go looking into tbb/lib so we move what's in there to libs
-            # and symlink the right lib from /tbb/libs/intel64/... to lib
```
Comment (boegel) [inline easybuild/easyblocks/t/tbb.py:165]: @mboisson Why remove this comment here, there's symlinking being done below?

The comment should state it's creating a `tbb/lib` symlink though?

File: easybuild/easyblocks/t/tbb.py [hunk 2]
```diff
@@ -171,9 +170,13 @@ def install_step(self):
             cand_lib_paths = glob.glob(os.path.join(self.installdir, 'build', '*_release'))
             if len(cand_lib_paths) == 1:
                 self.libpath = os.path.join('build', os.path.basename(cand_lib_paths[0]))
+                # and symlink the right lib from /tbb/libs/intel64/... to lib
```
Comment (boegel) [inline easybuild/easyblocks/t/tbb.py:173]: This comment is incorrect: the `and` start doesn't make sense, and it symlinks to a subdirectory of `build`?

**Comment by mboisson** [comment]
@boegel, see if the comments make more sense now.

**Comment by akesandgren** [comment]
@boegel ping...

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
tested & approved, thanks @mboisson!

**Pull request merged**